test/new-e2e/tests/otel/otlp-ingest/receive_resource_spans_v2_test.go

```diff
@@ -36,7 +35,9 @@ datadog:
     containerCollectUsingFiles: false
 `
 	t.Parallel()
-	e2e.Run(t, &otlpIngestSpanReceiverV2TestSuite{}, e2e.WithProvisioner(awskubernetes.KindProvisioner(awskubernetes.WithAgentOptions(kubernetesagentparams.WithHelmValues(values)))))
+	e2e.Run(t, &otlpIngestSpanReceiverV2TestSuite{}, e2e.WithProvisioner(
```

Comment:
Nit: similarly to @agagniere comment above we should probably aim to have consistent styling; also the alternate style helps with readability.